### PR TITLE
Fix test_merge_tree_s3_failover with debug build

### PR DESCRIPTION
**Pull request by ianton-ru**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix test_merge_tree_s3_failover with debug build

Detailed description / Documentation draft:
Test test_merge_tree_s3_failover relies on specific number of requests to S3 storage. In debug build there are additional request (from https://github.com/ClickHouse/ClickHouse/blob/master/src/Storages/MergeTree/MergeTreeDataPartWriterWide.cpp#L554) and test was broken.

### CODE REVIEW

File: tests/integration/test_merge_tree_s3_failover/test.py [hunk 1]
```diff
@@ -68,17 +68,19 @@ def drop_table(cluster):
 # S3 request will be failed for an appropriate part file write.
 FILES_PER_PART_BASE = 5  # partition.dat, default_compression_codec.txt, count.txt, columns.txt, checksums.txt
 FILES_PER_PART_WIDE = FILES_PER_PART_BASE + 1 + 1 + 3 * 2  # Primary index, MinMax, Mark and data file for column(s)
+FILES_PER_PART_WIDE_DEBUG = 2  # Additional requests to S3 in debug build
```
Comment (alexey-milovidov) [inline tests/integration/test_merge_tree_s3_failover/test.py:71]: But there is no explanation of the logic of this change...

**Review by alexey-milovidov — APPROVED** [review]

File: tests/integration/test_merge_tree_s3_failover/test.py (hunk 1 above)
Comment (alexey-milovidov) [reply tests/integration/test_merge_tree_s3_failover/test.py:73]: I've added it.

**Comment by alexey-milovidov** [comment]
@qoega Could you please enable integration tests with debug builds?

**Comment by qoega** [comment]
> @qoega Could you please enable integration tests with debug builds?

https://a.yandex-team.ru/review/1936442

**Pull request merged**